timesketch/lib/analyzers/phishy_domains.py

```diff
@@ -41,8 +41,17 @@ def __init__(self, index_name, sketch_id):
 
         self.domain_scoring_threshold = current_app.config.get(
             'DOMAIN_ANALYZER_WATCHED_DOMAINS_SCORE_THRESHOLD', 0.75)
-        self.domain_scoring_whitelist = current_app.config.get(
-            'DOMAIN_ANALYZER_WHITELISTED_DOMAINS', [])
+        self.domain_scoring_exclude_domains = current_app.config.get(
+            'DOMAIN_ANALYZER_EXCLUDE_DOMAINS', [])
+
+        # TODO: remove that after a 6 months, this following check is to ensure
+        # compatibility of the config file.
+        if len(self.domain_scoring_exclude_domains) == 0:
+            print("Warning, DOMAIN_ANALYZER_WHITELISTED_DOMAINS has been deprecated. "
```

Comment:
please use single quotes, keep the consistency (' vs ")

Reply:
done